src/style.css

```diff
@@ -1 +1,74 @@
 /* styles go here */
+.form_field {
+  display: block;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.fieldset-1 {
```

Comment:
You don't need such a lot of code in styles. Just add the correct margin to the fieldsets and to the labels. Use common classes for it.